### DISCUSSION TITLE
rebalancing all coolers in an .mcool

### DISCUSSION THREAD

**Discussion by golobor** [open2c/cooler#413] [Q&A] [answered]
I keep forgetting - how does one balance all coolers in a multi-cooler?
...is there any change `cooler balance` could be extended to assist users in this (common) task?

**Comment by nvictus** [comment, marked as answer]
```sh
for i in `cooler ls foo.mcool`; do
    cooler balance {options} $i;
done
 ```
 > any change cooler balance could be extended to assist users in this (common) task?
 
Since all `cooler` commands operate on single coolers and this task needs to be done sequentially anyway, `cooler ls` is really the concept to teach. 

Extending commands to operate on mcools or larger collections opens up a lot of surface area and needs some thought. For now, probably better to better document or make this answer more visible.
